File: api-artist-workspace/src/main/java/org/example/workspace/mapper/UserMapper.java
package org.example.workspace.mapper;

import org.example.workspace.dto.response.UserResDto;
import org.example.workspace.entity.User;
import org.mapstruct.Mapper;

@Mapper(componentModel = "spring", uses = {UserSnsMapper.class})
public interface UserMapper extends BaseMapper<User, UserResDto> {
}

File: level-up-backend-gpt/level2-book-management-system/src/main/kotlin/org/example/level2bookmanagementsystem/base/mapper/UserMapper.kt
package org.example.level2bookmanagementsystem.base.mapper

import org.example.level2bookmanagementsystem.base.dto.response.UserResponse
import org.example.level2bookmanagementsystem.base.dto.response.UserSimpleResponse
import org.example.level2bookmanagementsystem.security.command.SignupCommand
import org.example.level2bookmanagementsystem.security.entity.TbUser
import org.mapstruct.Mapper

@Mapper(componentModel = "spring", config = BaseMapper::class)
interface UserMapper: BaseMapper<SignupCommand, TbUser, UserResponse, UserSimpleResponse> {

}